transport.go

```diff
@@ -369,9 +383,7 @@ func (t *Transport) maybeSendStatelessReset(p *receivedPacket, connID protocol.C
 	rand.Read(data)
 	data[0] = (data[0] & 0x7f) | 0x40
 	data = append(data, token[:]...)
-	if _, err := t.conn.WritePacket(data, p.remoteAddr, p.info.OOB()); err != nil {
```

Comment:
`maybeSendStatelessReset` is still called in a separate Go routine. I'm wondering if the send queue Go routine should be the one that's composing the packet though, that would free up the Go routine that's reading from the connection.

Reply:
fixed the bug. 
In case we do the computation in the sendQueue goroutine, do you think it makes sense to spawn multiple such go routines?

Reply:
No, doing it in a single Go routine is fine. We don't expect to be sending many stateless resets under normal circumstances.